nixops/resources/aws_vpn_connection.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+
+from nixops.state import StateDict
+from nixops.diff import Diff, Handler
+import nixops.util
+import nixops.resources
+from nixops.resources.ec2_common import EC2CommonState
+import nixops.ec2_utils
+
+class AWSVPNConnectionDefinition(nixops.resources.ResourceDefinition):
+    """Definition of an AWS VPN connection."""
+
+    @classmethod
+    def get_type(cls):
+        return "aws-vpn-connection"
+
+    @classmethod
+    def get_resource_type(cls):
+        return "awsVPNConnections"
+
+    def show_type(self):
+        return "{0}".format(self.get_type())
+
+class AWSVPNConnectionState(nixops.resources.ResourceState, EC2CommonState):
+    """State of a AWS VPN gateway."""
+    state = nixops.util.attr_property("state", nixops.resources.ResourceState.MISSING, int)
+    access_key_id = nixops.util.attr_property("accessKeyId", None)
+    _reserved_keys = EC2CommonState.COMMON_EC2_RESERVED + ["vpnConnectionId"]
+
+    def __init__(self, depl, name, id):
+        nixops.resources.ResourceState.__init__(self, depl, name, id)
+        self._state = StateDict(depl, id)
+        self._client = None
+        self.handle_create_vpn_conn = Handler(['region', 'vpnGatewayId', 'customerGatewayId', 'staticRoutesOnly'], handle=self.realize_create_vpn_conn)
+        self.handle_tag_update = Handler(['tags'], after=[self.handle_create_vpn_conn], handle=self.realize_update_tag)
```

Comment:
So I noticed that tags don't get updated, but it's not clear to me why. Shouldn't handle_tag_update trigger whenever the tags attribute changes?

Reply:
@edolstra hm weird, thought It was fixed. I'll double check.

Reply:
Well, that was last week, so maybe it works now...